src/charts/time_series_labeler.coffee

```diff
@@ -46,7 +59,7 @@ Ember.Charts.TimeSeriesLabeler = Ember.Mixin.create
     secondIndex = _.findIndex(allTicks, findTick(labelledTicks[1]))
     firstIndex = _.findIndex(allTicks, findTick(labelledTicks[0]))
     secondIndex - firstIndex - 1
-  .property 'xDomain', 'selectedInterval', 'labelledTicks'
+  .property 'xDomain', 'xAxisDomainType', 'labelledTicks'
```

Comment:
Don't this computed property depend on `'dynamicXAxis', 'maxNumberOfLabels'` also?